tests/view_tests/tests/test_debug.py

```diff
@@ -417,7 +417,7 @@ def test_reporting_frames_without_source(self):
         html = reporter.get_traceback_html()
         self.assertIn('generated in funcName', html)
         text = reporter.get_traceback_text()
-        self.assertIn('"generated" in funcName', text)
+        self.assertIn('"generated", line 2, in funcName', text)
```

Comment:
We want to change whitespaces in the traceback, so we should test these changes e.g.
```python
        self.assertIn(
            '  File "generated", line 2, in funcName\n'
            '    <source code not available>',
            text,
        )
```